tc-server/src/main/java/com/tc/objectserver/entity/InvokeContextImpl.java

```diff
@@ -46,17 +46,6 @@ public InvokeContextImpl(ClientSourceId sourceId, int concurrencyKey, long oldes
     this.concurrencyKey = concurrencyKey;
     this.oldestid = oldestid;
     this.currentId = currentId;
-    setThreadLocal();
-  }
-
-  private void setThreadLocal() {
-    if (this.sourceId.isValidClient()) {
-      INHERITED.set(this);
```

Comment:
Any reason to not also delete the field itself?

Reply:
yeah.  oversight